sql_runner/snowflake_target.go

```diff
@@ -28,8 +28,9 @@ import (
 
 // Specific for Snowflake db
 const (
-	loginTimeout  = 5 * time.Second                // by default is 60
-	multiStmtName = "multiple statement execution" // https://github.com/snowflakedb/gosnowflake/blob/e909f00ff624a7e60d4f91718f6adc92cbd0d80f/connection.go#L57-L61
+	snowplowAppName = `Snowplow_OSS`
```

Comment:
I think we should check with the partnership team to find out what they'd like the value to be here.

Since we run SQL-runner as part of our non-OS offering, I wonder if we'd like to have a way to distinguish them.

One option here is to hardcode `Snowplow `, then allow the target configuration to add a value to this string.

@paulboocock do we have any specification on this? Do we know if there are any rules to how attribution is done on the other side, that we should work to?

Reply:
Hey, apologies for not better defining this. This value should be configurable. It should _always_ start with `Snowplow_` which should not be configurable, then we can append a custom string to the end, e.g. `Snowplow_OSS` as the default. Then we will use a different string when we run this for customers.

cc @TiganeteaRobert